README.md

```diff
@@ -89,12 +89,7 @@ as enabling use_thrift.
 
 ```python
 return lightstep.Tracer(
-    component_name='WhatIDoWhatIDidnt',
-    access_token='your-token-here',
-    collector_host='satellite.host',
-    collector_port=80,
-    collector_encryption='none',
-    verbosity=2,
+    ...
```

Comment:
I remember @austinlparker mentioning this one, but I feel we are mixing a few things here (simplifying the README with the access-token part). Lets go this way for now, though.